doc/steps.md

```diff
@@ -571,7 +573,23 @@ Output facts:
 
 ### Step `backup_stop`
 
-Stop started [backup](/doc/backup.md) on the instance.
+Stop started [backup](/doc/backups.md) on the instance.
+
+### Step `restore`
+
+[Restore](/doc/backups.md#how-to-restore-from-saved-backup) each instance from a [backup](#step-backup) archive.
+
+Input variables from config:
+
+- `cartridge_paths_to_keep_before_restore` - list of full paths or relative paths
+  to work/memtx/vinyl/wal directory that should be kept on instance restore
```

Comment:
Let's use \`\` for `work/memtx/vinyl/wal`.

list of paths that are absolute or relative to ... ?